### DISCUSSION TITLE
Provide the user information about where they're located in the comment thread

### DISCUSSION THREAD

**Discussion by Porkepix** [Docile-Alligator/Infinity-For-Reddit#918] [Ideas]
It's currently impossible for users to figure out where in the comment thread are they currently.

Whether is it with scrollbar or with an indicator of `comment XX / YY total comments`, it would be nice to knew where are we. Bonus points if it's possible to quickly move through the thread (again, scrollbar, or setting the comment number)

**Comment by Docile-Alligator** [comment]
Good suggestions.
